lib/sqlalchemy/sql/util.py

```diff
@@ -1164,7 +1164,7 @@ def replace(
                         return None
                 return self.selectable  # type: ignore
             elif isinstance(col, Alias) and isinstance(
-                col.element, TableClause
+                col.element, TableClause  # type: ignore[attr-defined]
```

Comment:
**mike bayer** (zzzeek) wrote:

remove

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4325